test/unit/data/transfers/{id}/error.js

```diff
@@ -1,5 +1,5 @@
 'use strict'
-var Mockgen = require('../../mockgen.js')
+var Mockgen = require('../../mockgen')
```

Comment:
Should this be `let` or `const`?

Reply:
I have changed this, but note that these are auto-generated by swag plugin

Reply:
Ah, I see. That makes sense.